evaluator/evaluator/routers/dataset.py

```diff
@@ -1,22 +1,260 @@
 import logging
-from typing import List
+from typing import Dict, List
 
-from fastapi import APIRouter
+from bson import ObjectId
+from evaluate import load
+from fastapi import APIRouter, Body, Depends, FastAPI, HTTPException, Request
+from square_auth.auth import Auth
 
-from evaluator.models import DataSet
+from evaluator import mongo_client
+from evaluator.core.dataset_handler import DatasetDoesNotExistError, DatasetHandler
+from evaluator.models import Dataset, DataSet
+from evaluator.mongo.mongo_client import MongoClient
+from evaluator.routers import client_credentials
+from evaluator.routers.evaluator import get_dataset_metadata
 
 logger = logging.getLogger(__name__)
-
 router = APIRouter(prefix="/dataset")
+auth = Auth()
+
+valid_skill_type = {
```

Comment:
Konstanten generell Groß-schreiben. Und sollte vermutlich in `models.py`.
Wieso ist das key-value? Wenn ich das richtig sehe, verwendest du nie die values aus dem Dictionary 🤔 dann braucht es auch kein dict sein.

Reply:
habe ich gemacht